Library/ExternalProduct.cs

```diff
@@ -0,0 +1,181 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+
+    /// <summary>
+    /// Represents external_products for accounts
+    /// </summary>
+    public class ExternalProduct : RecurlyEntity
+    {
+
+
+
+        public string Name { get; private set; }
+        public string PlanName { get; private set; }
```

Comment:
It looks like the top level non-nested nodes on an external_product are `name, created_at, updated_at` and the nested nodes are `plan` and `external_product_references`.  Is it an issue for this class that planName and PlanCode are directly on the `ExternalProduct`?

Reply:
That's a good point. For some reason I wasn't thinking and didn't think to follow the same way it's done in Subscription and give the user access to the whole plan.

Reply:
Follows the same idea as Subscription that ultimately puts a PlanCode on the top level, but you can now call `subscription.Plan` and get everything